crates/sui-bridge-indexer/src/migrations/00000000000000_diesel_initial_setup/up.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE token_transfer_data
```

Comment:
should we have more info into this table so that it can be queried on its own?
for instance we have no timestamp which makes time query only possible with a `join` (e.g. highest amount in the past week).
Also maybe sender and recipient address are more interesting indexes?